### ISSUE TITLE
Draft Notices for -80s 

### ISSUE THREAD

**Issue by sr320** [sr320/LabDocs#584]
Draft "sign" of -80s describing what why are and rules and regulations for opening :)
Also emergency contact. Etc

Provide url with editable doc or markdown here.

**Comment by grace-ac** [comment]
Here's a draft! Probably too wordy. Link: [here](https://docs.google.com/a/uw.edu/document/d/1rlGukWdNGY12HwR8U_efonUTWpcPPFf1omnpVZ119o4/edit?usp=sharing)

**Comment by sr320** [comment]
Add sam contact info too.
and phone # for me - 206-866-5141

Go ahead and print one up. Big font. Lets take a look.
